modules/templates/templates/create-template.tsx

```diff
@@ -0,0 +1,286 @@
+import React, { useState } from "react";
```

Comment:
Im not sure why this component is modified (I think is renamed?)

We dont need a template modal

1 - On creating template we just create a default template
2 - On edit template (pencil) we go to the template page
3 - On the template detail page we have an option to edit the title (A little pencil near the title which makes the title into editable)
4 -On the template detail page we need to give error for dupplicated field names, don't let them save if it is duplicated.


I dont think this change is making any of the required changes, as this was not the file to be modified